docs/recipes/custom-connector.md

```diff
@@ -0,0 +1,90 @@
+## Introduction
+
+Custom wallet connectors allow users to connect their preferred wallets to your decentralized application (dApp). This provides flexibility and improves the user experience by giving users options beyond popular default wallets like MetaMask.
+
+
+## Install Dependencies
+First, install the necessary packages:
+
+```
+npm install @scaffold-stark/stark-burner
+```
+OR
+
+```
+yarn add @scaffold-stark/stark-burner
+```
```

Comment:
I meant breaking down the burner wallet as an example connector ser. not installing it, this would defeat the purpose of the documentation